games-emulation/dosbox-staging/dosbox-staging-0.75.0.ebuild

```diff
@@ -16,7 +16,10 @@ IUSE="alsa debug dynrec opengl opus"
 RDEPEND="alsa? ( media-libs/alsa-lib )
 	debug? ( sys-libs/ncurses:0= )
 	opengl? ( virtual/opengl )
-	opus? ( media-libs/opus )
+	opus? (
+		media-libs/opus
+		media-libs/opusfile
```

Comment:
Thanks for testing that! Looking at [configure.ac](https://github.com/dosbox-staging/dosbox-staging/blob/v0.75.0/configure.ac#L500) opusfile is actually the needed dep for dosbox-staging, opus itself comes as dependency of opusfile. So we can drop it from deps here

Reply:
Fixed